src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -1400,6 +1400,8 @@
     DEPORTATION_ORDER_OPTIONS(
         "deportationOrderOptions", new TypeReference<YesOrNo>(){}),
 
+    CCD_REFERENCE_NUMBER_FOR_DISPLAY(
```

Comment:
I would consider calling the `CCD_CASE_ID` or `CCD_CASE_ID_FOR_DISPLAY` for better accuracy.

Reply:
Left as CCD_REFERENCE_NUMBER_FOR_DISPLAY